QuadPaySDK/PaymentWidget/PaymentWidget.swift

```diff
@@ -214,6 +253,11 @@ extension PaymentWidget {
 
         //Passed parameters to the header for pop up
         //{minModal, merchantId,isMFPPMerchant, learnmoreURL}
+        if(learnMoreUrl != ""){
+            if(!learnMoreUrl.contains("https://")){
+                learnMoreUrl = "https://" + learnMoreUrl
+            }
+        }
```

Comment:
Same comment as on Android, we could have a URL helper class which takes in a string and checks if it has the scheme (which can be a public static property on the class).

Reply:
Created a helper class